helpers/database.php

```diff
@@ -107,3 +107,218 @@ function getStockItemImage($id, $databaseConnection)
 
     return $R;
 }
+
+
+
+function abbreviate($string){ # maakt een afkorting van de ingevoerde waarde
+    $abbreviation = "";
+    $string = ucwords($string);
+    $words = explode(" ", "$string");
+    foreach($words as $word){
+        $abbreviation .= $word[0];
+    }
+    return $abbreviation;
+}
+function getNewStateProvinceID($databaseConnection)
+{
+    $Query = "
+    SELECT max(StateProvinceID)
+    FROM stateprovinces";
```

Comment:
Ik heb mijn twijfels of het nodig is om op deze manier de new op te halen dit geld voor alle plekken waar dit met een max gebeurt